File: src/movierental/StatementDemo.java
package movierental;

/**
 * Demo the use of Customer statement function.
 */
public class StatementDemo {
	public static void main(String[] args) {
		Movie[] movies = new Movie[] {
				new Movie("Bridge of Spies", Movie.NEW_RELEASE),
				new Movie("CitizenFour", Movie.REGULAR),
				new Movie("Frozen", Movie.CHILDRENS),
				new Movie("El Camino", Movie.NEW_RELEASE),
				new Movie("Particle Fever", Movie.REGULAR)
		};
		Customer cust = new Customer("Edwin Snowden");
		for(Movie m: movies) {
			cust.addRental(new Rental(m, days));
			days++;
		}
		System.out.println( cust.createStatement() );
	}
}

File: statementdemo.py
from movie import Movie
from rental import Rental
from customer import Customer

def make_movies():
	movies = [
		Movie("The Irishman", Movie.NEW_RELEASE),
		Movie("CitizenFour", Movie.REGULAR),
		Movie("Frozen", Movie.CHILDRENS),
		Movie("El Camino", Movie.NEW_RELEASE),
		Movie("Particle Fever", Movie.REGULAR)
		]
	return movies

def create_customer_and_rental():
	customer = Customer("Edward Snowden")
	days = 1
	for movie in make_movies():
		customer.addRental(Rental(movie,days))
		days += 1
	return customer

def print_report(customer: Customer):
	print( customer.createStatement() )

if __name__ == '__main__':
	customer = create_customer_and_rental()
	print_report(customer)
